docs/tutorials/mlflow.ipynb

```diff
@@ -0,0 +1,781 @@
+{
```

Comment:
<p>maybe we can do "clear output" on this cell, to hide these warnings regarding the versions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/4694/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>That would be a good idea. Even edit the file manually to remove this line.</p>